backend/spec/requests/memos_spec.rb

```diff
@@ -28,4 +28,48 @@
       end
     end
   end
+
+  describe 'PUT /memos/:id' do
+    context 'コンテンツが有効な場合' do
+      let(:existing_memo) { create(:memo) }
+      let(:update_memo_params) { { content: '新しいコンテンツ' } }
+
+      it 'memoが更新され、204になる' do
+        aggregate_failures do
+          put "/memos/#{existing_memo.id}", params: { memo: update_memo_params }
+          expect(response).to have_http_status(:no_content)
+          existing_memo.reload
+          expect(existing_memo.content).to eq('新しいコンテンツ')
+        end
+      end
+    end
+
+    context 'バリデーションエラーになる場合' do
+      let(:existing_memo) { create(:memo) }
+      let(:invalid_params) { { content: '' } }
+
+      it '422になり、エラーメッセージがレスポンスとして返る' do
+        aggregate_failures do
+          put "/memos/#{existing_memo.id}", params: { memo: invalid_params }
+          existing_memo.reload
+          expect(response).to have_http_status(:unprocessable_entity)
+          expect(response.parsed_body['errors']).to eq(['コンテンツを入力してください'])
+        end
+      end
+    end
+  end
+
+  context 'タイトルを更新しようとした場合' do
+    let(:existing_memo) { create(:memo) }
+    let(:invalid_params) { { title: '新しいタイトル' } }
```

Comment:
invalid_というprefixはなくても、contextを見ればinvalidなparamsを入れていることは想起できるかなと思いました〜
```suggestion
    let(:params) { { title: '新しいタイトル' } }
```

Reply:
おっしゃる通りだと思いました。
無条件に詳細な命名をするのではなく、状況に合わせて必要かどうか考えること意識していきます！